examples/pytorch/deepergcn/models.py

```diff
@@ -0,0 +1,121 @@
+import torch.nn as nn
+import torch.nn.functional as F
+import dgl.function as fn
+
+from ogb.graphproppred.mol_encoder import AtomEncoder
+from dgl.nn.pytorch.glob import SumPooling, AvgPooling, MaxPooling
+from modules import norm_layer
+from layers import GENConv
+
+
+class DeeperGCN(nn.Module):
+    r"""
+
+    Description
+    -----------
+    Introduced in `DeeperGCN: All You Need to Train Deeper GCNs <https://arxiv.org/abs/2006.07739>`_
```

Comment:
Strictly speaking, the reason we added "``_" is for the purpose of doc generation with sphinx. Without the use of that, we do not need these.